sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -148,16 +148,16 @@ def test_equal_similarities_and_preferences():
     X = np.array([[0, 0], [1, 1], [-2, -2]])
     S = -euclidean_distances(X, squared=True)
 
-    assert_false(_equal_similarities_and_preferences(S, np.array(0)))
-    assert_false(_equal_similarities_and_preferences(S, np.array([0, 0])))
-    assert_false(_equal_similarities_and_preferences(S, np.array([0, 1])))
+    assert not(_equal_similarities_and_preferences(S, np.array(0)))
+    assert not(_equal_similarities_and_preferences(S, np.array([0, 0])))
+    assert not(_equal_similarities_and_preferences(S, np.array([0, 1])))
```

Comment:
I know it's harder with a regexp, but if we do this properly, we would need to remove the parenthesis,
```py
assert not _equal_similarities_and_preferences(S, np.array(0))
```